domino_cost/cost.py

```diff
@@ -132,44 +156,56 @@ def workload_cost_details(cost_table: DataFrame) -> DataTable:
         "prefix": None,
         "specifier": "$,.2f",
     }
-    table = dash_table.DataTable(
-        columns=[
-            {"name": "TYPE", "id": "TYPE"},
-            {"name": CostLabels.PROJECT_NAME.value, "id": CostLabels.PROJECT_NAME.value},
-            {"name": CostLabels.BILLING_TAG.value, "id": CostLabels.BILLING_TAG.value},
-            {"name": CostLabels.USER.value, "id": CostLabels.USER.value},
-            {"name": "START DATE", "id": "FORMATTED START"},
-            {
-                "name": CostFieldsLabels.CPU_COST.value,
-                "id": CostFieldsLabels.CPU_COST.value,
-                "type": "numeric",
-                "format": formatted,
-            },
-            {
-                "name": CostFieldsLabels.GPU_COST.value,
-                "id": CostFieldsLabels.GPU_COST.value,
-                "type": "numeric",
-                "format": formatted,
-            },
-            {
-                "name": CostFieldsLabels.STORAGE_COST.value,
-                "id": CostFieldsLabels.STORAGE_COST.value,
-                "type": "numeric",
-                "format": formatted,
-            },
+    display_cost_col = True if cost_table[CostAggregatedLabels.CLOUD_COST.value].sum() > 0 else False
```

Comment:
In the future we could probably have a boolean check as a global variable and if `get_cloud_cost_sum` finds cloud costs are not enabled, it can switch that variable directly to `False`